js/ssai_pulse.js

```diff
@@ -574,11 +574,37 @@ OO.Ads.manager(function(_, $)
      */
     var _appendAdsProxyQueryParameters = _.bind(function(url, adId)
     {
-      // vastUrl + '&offset=5&aid=' + adid
-      return url + OFFSET_PARAM + OFFSET_VALUE + AD_ID_PARAM + adId;
+      var offset = OFFSET_PARAM + OFFSET_VALUE;
+      var newUrl = _appendParamToUrl(url, offset);
+
+      var adIdParam = AD_ID_PARAM + adId;
+      newUrl = _appendParamToUrl(url, adIdParam);
```

Comment:
I think appendParamToUrl should be taking in the newUrl here, as opposed to url.